docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+| `Copyright`                 | `Copyright`                  | `copyright`                 | Copyright details for the package. language.                                                                      |
+| `Licensing - Expression`    | `PackageLicenseExpression`   | `license type="expression"` | An SPDX license expression or path to a license file within the package, often shown in UIs like NuGet.org.       |
+| `Licensing - File`          | `PackageLicenseFile`         | `license type="file"`       | File for custom license that isn't supported by license expressions                                               |
+| `Project URL`               | `PackageProjectUrl`          | `projectUrl`                | A URL for the project homepage.                                                                                   |
+| `Icon File`                 | `PackageIcon`                | `icon`                      | An image to use as the icon for the package.                                                                      |
+| `Repository URL`            | `RepositoryUrl`              | `repository url=""`         | URL to the repository from which the package was built.                                                           |
+| `Repository type`           | `RespositoryType`            | `repository type=""`        | Type of repository the repository URL is pointing to (i.e. git)                                                   |
+| `Tags`                      | `PackageTags`                | `tags`                      | A space-delimited list of tags and keywords that describe the package. Tags are used when searching for packages. |
+| `Release notes`             | `PackageReleaseNotes`        | `releaseNotes`              | A description of the changes made in this release of the package.                                                 |
+
+### Package ID
+
+If you're publishing a completely new package:
+
+✔️ DO choose a package ID that is unique and clearly differentiated from existing packages on NuGet.org.
+> You can make sure a package ID is unique and differntiable by searching for the ID on NuGet.org or checking if the following link exists: https://www.nuget.org/packages/<package name\>.
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria).
+> Reserving the prefix ID for your package will let you get the verified check mark:
+> ![image](media/Verified-check-mark.png)
+> Check out the [Package ID prefix reservation docs](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation) to learn more.
+
+### Package Version
+
+✔️ CONSIDER using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
```

Comment:
I would recommend just saying "Semantic Versions (SemVer)" without specifying 2.0.0 at first. There is a lot of other references in docs to SemVer 2.0.0 in light of how it is different than SemVer 1.0.0 (dots in pre-release and build metadata). Consider mentioning something like "NuGet supports the SemVer 2.0.0 specification, however some older clients may only support SemVer 1.0.0" with a link to the distinctions.

Reply:
Just curious, what's the value in this recommendation? Off-hand this appears superfluous, should we remove it?

Reply:
My understanding was that SemVer is a common industry standard, although not universally adopted on NuGet.org. Doesn't encouraging SemVer help created shared understanding around versioning meaning (i.e. Major.Minor.Patch)? I'm far from an expert though, so I'm happy to hear any specific critiques 😊

Reply:
I am a bit bit confused with "CONSIDER". If these are best practices guide, shouldn't we just say "Use.."? Why leave the iota of doubt? 😄. The rationale and explanation should be enough for authors to decide whether its something they should use or not.

Reply:
I think this should be `Do` as well. 

Have given collapsible sections for explanations a thought? Markdown is pretty limiting as far as that goes and I have no  clue if it's a docs recommendation.

Reply:
What's our goal in recommending semver? Do we want people to version their packages properly to declare breaking changes, new features, and bug fixes? If so, we should clarify the "why" of semver. If not, what's the value in recommending semver? That folks avoid using 4 digits to represent their versions? That doesn't seem all that useful.